src/stream/manager.rs

```diff
@@ -78,31 +81,41 @@ pub fn add_stream_and_start(
         .clone();
 
     let mut stream = stream_backend::new(&video_and_stream_information)?;
-    let mavtype: mavlink::common::VideoStreamType = match &stream {
-        StreamType::UDP(_) => mavlink::common::VideoStreamType::VIDEO_STREAM_TYPE_RTPUDP,
-        StreamType::RTSP(_) => mavlink::common::VideoStreamType::VIDEO_STREAM_TYPE_RTSP,
-        StreamType::REDIRECT(video_strem_redirect) => match video_strem_redirect.scheme.as_str() {
-            "rtsp" => mavlink::common::VideoStreamType::VIDEO_STREAM_TYPE_RTSP,
-            "mpegts" => mavlink::common::VideoStreamType::VIDEO_STREAM_TYPE_MPEG_TS_H264,
-            "tcp" => mavlink::common::VideoStreamType::VIDEO_STREAM_TYPE_TCP_MPEG,
-            "udp" | _ => mavlink::common::VideoStreamType::VIDEO_STREAM_TYPE_RTPUDP,
-        },
-    };
 
-    stream.mut_inner().start();
-    manager.streams.push(Stream {
-        stream_type: stream,
-        video_and_stream_information: video_and_stream_information.clone(),
-        mavlink_camera: MavlinkCameraHandle::new(
+    let mavlink_camera_handle = if let StreamType::WEBRTC(_) = &stream {
+        None
```

Comment:
You can use `webrtc://<ice-user>:<ice-password>@<Address/Port>?<parameters>` for the video uri in mavlink manager.

Reply:
I didn't get the idea, I'd looked for any webrtc thing in mavlink and couldn't find anything, can you provide a reference of that?

Reply:
You can add an url for the mavlink camera manager and not use none, having a mavlink camera handle for the webrtc stream.

Reply:
Oh, I still need to do this part. Because webrtc definition is a combination of stun, turn, and signalling servers, how are we going to send it? I haven't checked yet, but we might only need the signalling server to be able to receive the stream somewhere else.

Reply:
We could send only one of them, the first one provided.